migrations/sql/V2019.05.02.06.01__add_dangerous_occurrence_code_subsections.sql

```diff
@@ -0,0 +1,59 @@
+ALTER TABLE compliance_article ADD COLUMN long_description text;
```

Comment:
Apologies for being late to the party... if there is still an opportunity, can you please change the data type from 'text' to 'varchar' and provide a max length?

Reply:
What max length will be required?

Reply:
Depends on how much we think this column will contain... I just checked quickly, and the longest long_description value being added for this release is about 310 characters. We could set the max for 400 characters... but could set it at 500 for extra wiggle room.

Reply:
Column is now a varchar 500